internal/register/register.go

```diff
@@ -0,0 +1,28 @@
+package register
+
+import (
+	"file-storage/internal/database"
+	"log"
+	"github.com/labstack/echo/v4"
+	"net/http"
+	"file-storage/internal/userstruct"
+)
+
+type RegisterRequest struct {
+	Login string `json:"email"`
+	Password string `json:"password"`
+}
+
+func RegisterUser(c echo.Context) error{
+	request := new(RegisterRequest)
```

Comment:
зачем через new? можно просто структуру пустую
```suggestion
	request := RegisterRequest{}
```